local-cli/link/link.js

```diff
@@ -49,7 +49,8 @@ const linkDependency = (platforms, project, dependency) => {
         return null;
       }
 
-      const isInstalled = linkConfig.isInstalled(project[platform], dependency.config[platform]);
+      const isInstalledParameter = platform == "android" ? dependency.name : dependency.config[platform];
+      const isInstalled = linkConfig.isInstalled(project[platform], isInstalledParameter);
```

Comment:
That makes sense, although I would be much more in favor of having it use the same signature. 

CC: @Kureev

Reply:
The thing is `dependency.config[platform]` does not contain `dependency.name`, should we add `dependencyName` to `dependency.config[platform]` object then?

Reply:
Unlink suffers the same issue for android and iOS unlink is broken

Reply:
Let's just add the `name` parameter to the `isInstalled` signature.